serverless/pages/what-is-elasticsearch-serverless.asciidoc

```diff
@@ -9,11 +9,11 @@ preview:[]
 
 Elasticsearch allows you to build custom applications. Whether you have structured or unstructured text, numerical data, or geospatial data, Elasticsearch can efficiently store and index it in a way that supports fast searches.
 
-.Understanding Elasticsearch on serverless
-[IMPORTANT]
-====
-Refer to <<elasticsearch-differences,Serverless differences>> and <<elasticsearch-technical-preview-limitations,Technical preview limitations>> for important details, including features and limitations specific to {es} on serverless.
-====
+// .Understanding Elasticsearch on serverless
```

Comment:
This page is being updated and once https://github.com/elastic/docs-content/pull/183 is merged, I'll update the branch